app/models/review.rb

```diff
@@ -0,0 +1,5 @@
+class Review < ApplicationRecord
+  belongs_to :application
+
+  validates :rating, presence: true
```

Comment:
Rating should probably have: 
`  validates :rating, inclusion: { in: [0, 1, 2, 3, 4, 5] }
  validates :rating, numericality: { only_integer: true }`
though it depends on how exactly we want to handle it. This is how animal_kingdom did it though,